### ISSUE TITLE
Change line-break style to unix only

### ISSUE THREAD

**Issue by ayuhito** [privatenumber/eslint-config#33]
I'm currently using a Windows machine whilst the ESLint rule line-break style is dynamically set depending on the platform [here](https://github.com/privatenumber/eslint-config/blob/c0ae6b6c689d331f574bd0ba102386157cb5bf87/rules/style.js#L165). 

This isn't great since it actually wants me to then autofix all the code in your repos to CRLF when ideally we want to consistently stick with LF regardless of the platform. It basically blocks any developer using Windows to contribute to your repos. 😅 

Changing the line to `'linebreak-style': ['error', 'unix'],` will resolve the error.

**Comment by privatenumber** [comment]
To confirm, this is only happening on markdown files, right?

https://github.com/privatenumber/eslint-config/runs/7103699439?check_suite_focus=true

**Comment by ayuhito** [comment]
Here's an example of my IDE for a JS file:

<img width="713" alt="image" src="https://user-images.githubusercontent.com/38220115/176358514-61d4d2df-d43f-4231-add6-b21862ca9d3b.png">

Also [here](https://github.com/airbnb/javascript/blob/d8cb404da74c302506f91e5928f30cc75109e74d/packages/eslint-config-airbnb-base/rules/style.js#L178)'s a reference to the AirBnb style guide that I extend for my personal config. It's conventional to stick with one.

**Comment by privatenumber** [comment]
Interesting, I thought I had forked Airbnb's style for mine as well.

**Comment by ayuhito** [comment]
Yeah, the rabbit hole in airbnb/javascript#1224 makes it a little controversial. I actually think your solution isn't breaking, since I had my editor set to LF mode instead of CRLF. Switching it does remove the errors. 

It's pretty opinonated. If we stick with LF, I think it might be fair to ship an `.editorconfig` file with your projects so the IDE automatically chooses LF. It also could help with all the autoformatting indentation errors I've experienced since the project doesn't ship it's own Prettier config and/or EditorConfig. My VSCode formatter then goes to its own defaults which is very different from your indentation rules. Also switches everything from single quotes to double quotes in the process. 

I've had to disable my VSCode formatter (which uses Prettier under the hood) for the pkgroll repo to get around that.
